data/core/step4_install_rasa.md

```diff
@@ -0,0 +1,46 @@
+## Get started from website (including privacy clause etc)
+* get_started_step4
+    - action_greet_user
```

Comment:
same here -- i think this is just something that got copied, but i believe we want those in there

Reply:
Fixed - 3086274